avm/res/sql/server/key/main.bicep

```
@@ -8,14 +8,14 @@ param name string?
 @description('Conditional. The name of the parent SQL server. Required if the template is used in a standalone deployment.')
 param serverName string
 
-@description('Optional. The encryption protector type like "ServiceManaged", "AzureKeyVault".')
+@description('Optional. The server key type like \'ServiceManaged\', \'AzureKeyVault\'.')
```

```suggestion
@description('Optional. The server key type.')
```
See previous comment.